app/routes/events_.$eventId.tsx

```diff
@@ -26,6 +27,20 @@ export async function loader({ params }: LoaderFunctionArgs) {
   return json({ event });
 }
 
+export async function action({ params, request }: ActionFunctionArgs) {
+  const userSession = await requireUserSession(request);
+
+  if (!('userId' in userSession)) {
+    throw new Error('Unable to locate userId, please try logging in again.');
+  }
+  const event = await db.event.update({
```

Comment:
NIT: We could make sure the event doesn't contain the user already (just as a defensive measure).

Reply:
Captured in issue https://github.com/social-plan-it/plan-it-social-web/issues/136